src/lib/pci_virtio_sock.c

```diff
@@ -124,9 +124,9 @@ struct virtio_sock_hdr {
  * Debug printf
  */
 static int pci_vtsock_debug = 0;
-#define DPRINTF(params) do { if (pci_vtsock_debug) { printf params; fflush(stdout); } } while(0)
```

Comment:
there shouldn't be any need to change this debug code. this won't be enabled for release builds and there should be no need to redirect to a logger.